kcli/del-slb.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+set -xe
+
+oc apply -f del-slb-nncp.yaml
+./kcli/update-keepalived.sh enx525400f68001
```

Comment:
enx525400f68001

Is this a stable name?

Reply:
We are enforcing the mac address of primary and secondary nics per node and NetworkManager configure the interface with it https://github.com/RHsyseng/rhcos-slb/blob/simplify-networking/custom-config.fcc.tmpl#L11-L12, so it's stable name.

Reply:
I see, thanks for explaining